logger.go

```diff
@@ -0,0 +1,10 @@
+package checkup
+
+import (
+	"io/ioutil"
+	"log"
+)
+
+func init() {
+	log.SetOutput(ioutil.Discard)
```

Comment:
Surely we can move this one line into root.go or something?

Reply:
> I figured users would find output from this package undesirable unless they explicitly turn it on.

@parkr, doing this affects not only output from this package, but for all other packages using the standard logger of the `log` package. [`log.SetOutput`](https://godoc.org/log#SetOutput) is documented as:

> SetOutput sets the output destination for the standard logger.

Therefore:

```Go
package main

import "log"
import "github.com/sourcegraph/checkup"

func main() {
    log.Println("hello") // will get discarded because checkup did log.SetOutput(ioutil.Discard) during package init

    // ... use checkup
}
```

See #64 for more about this.